tools/install_macos_dependencies.sh

```diff
@@ -28,8 +28,12 @@ echo "Downloading libtensorflow..."
 curl https://storage.googleapis.com/tensorflow/libtensorflow/libtensorflow-cpu-darwin-x86_64-1.8.0.tar.gz > libtensorflow.tar.gz
 
 echo "Extracting and copying libtensorflow..."
-sudo tar zxf libtensorflow.tar.gz -C /usr/local
+TMP_DIR=`mktemp -d`
+tar zxf libtensorflow.tar.gz -C $TMP_DIR
```

Comment:
I suspect mktemp is not out got get us, but "$TMP_DIR" would make any spaces it might throw our way harmless (please change all instances).